doc/getting_and_using_mir.md

```diff
@@ -5,12 +5,11 @@ If you are not running Ubuntu things may not work as described below. While some
 applications are able to use Mir's Wayland support only Ubuntu carries "native"
 Mir support for GTK+, SDL2 and Qt applications and the Xmir X11 server. 
 
-Getting Mir examples
---------------------
+Mir was initially developed on and for Ubuntu. It is also available on Fedora. 
+We aim to make it available for other distributions, but currently you need to
+build it yourself see \ref getting_involved_in_mir.
```

Comment:
Some punctuation would be useful before "see", no? A comma, pause, or parenthesis?